monai/apps/auto3dseg/auto_runner.py

```diff
@@ -614,7 +617,15 @@ def run(self):
                 data_src_cfg_name=self.data_src_cfg_name,
             )
 
-            bundle_generator.generate(self.work_dir, num_fold=self.num_fold)
+            if self.gpu_customization:
```

Comment:
Hi @dongyang0122 , since you are adding a new way to use this feature in `AutoRunner`. Please update the doc-string in these two places:

- `kwargs` was mostly used for the `SaveImage` transform. Since the latest change is extending its usage. Please update the description of the argument
- Please also give an example code block as the other features in the `AutoRunner`.

Reply:
Also, I recommend creating a sperate set method for this new feature. Putting it in the `kwargs` of `__init__` lower the discoverability IMO.

Reply:
changes and test case are added.

Reply:
Hi @dongyang0122 , if the user did not call `set_gpu_customization`, then `self.gpu_customization` may be unassigned when running the `AutoRunner`. Please consider adding these in the `__init__`:
```python
self.gpu_customization = False
self.gpu_customization_specs = {}
```

Reply:
Please ignore my previous comment as it was resolved.